src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOperatorBase.java

```diff
@@ -0,0 +1,16 @@
+package se.liu.ida.hefquin.engine.queryplan.logical.impl;
+import se.liu.ida.hefquin.engine.queryplan.logical.LogicalOperator;
+
+public abstract class LogicalOperatorBase implements LogicalOperator {
+	private static int counter = 0;  
+    protected final int id;  
```

Comment:
Can you please be consistent about indenting either with spaces only or with tabs only (my preference is for tabs), but not a mix of both within the same file.